test/check-platform.js

```diff
@@ -43,6 +43,38 @@ t.test('os wrong (negation)', async t =>
 t.test('nothing wrong (negation)', async t =>
   checkPlatform({ cpu: '!enten-cpu', os: '!enten-os' }))
 
+t.test('nothing wrong with overridden os', async t =>
+  checkPlatform({
+    cpu: 'any',
+    os: 'enten-os',
+  }, false, {
+    os: 'enten-os',
+  }))
+
+t.test('nothing wrong with overridden cpu', async t =>
+  checkPlatform({
+    cpu: 'enten-cpu',
+    os: 'any',
+  }, false, {
+    cpu: 'enten-cpu',
+  }), { code: 'EBADPLATFORM' })
```

Comment:
```suggestion
  }))
```

Looks like a copy/paste leftover from a t.throws?

Reply:
@wraithgar You're right, thanks!